Path.json

```diff
@@ -564,6 +564,67 @@
 			]
 		},
 		{
+			"group": 2
+			"maxSpeed": 120
+			"twistingFactor": 0.2
+			"electrified": true
+			"objects": [
+    			{
+					"start": "KD",
+					"end": "KDV",
+					"length": 0
```

Comment:
FYI: Das Aufsplitten hätte ich an deiner Stelle erst gemacht, wenn der PR hier durch ist - in einem Follow-Up quasi.

Bei Streckenteilen mit Länge 0 muß man auch abwägen, was man drauf haben will und ggf. Zusammenfassen. Die Tests dürften auch fehlschlagen, wenn die Wegpunkte noch nicht in der Stations-Datei bekanntgemacht wurden.

Reply:
Hier würde ich aber auf 1 aufrunden - Düsseldorf Volksgarten ist ja ein richtiger Bahnhof.

@f2k1de Warum rundet tcroute bei Entfernung 0,9 eigentlich auf 0 ab?

Reply:
OK, ich werde es in einen separaten PR schreiben.

Reply:
Ja, ist vermutlich besser - dann hast du schon mal ein Erfolgserlebnis, wenn es gemerged ist und etwas später auch schon im Spiel landet.